tests/test_acceptance.py

```diff
@@ -84,93 +85,6 @@ def test_slurm(tmp_path: Path, scenario: Dict):
-    return template
-
-
-def create_real_future(result):
-    future = Future()
-    future.set_result(result)
-    return future
-
-
-def extract_unique_test_templates(test_templates):
-    unique_test_templates = {}
-    for test_template in test_templates:
-        template_name = test_template.name
-        if template_name not in unique_test_templates:
-            unique_test_templates[template_name] = test_template
-    return list(unique_test_templates.values())
-
-
-@patch("cloudai._core.base_installer.ThreadPoolExecutor", autospec=True)
-def test_install_success(mock_executor: Mock, slurm_system: SlurmSystem, test_template_success: Mock):
-    installer = BaseInstaller(slurm_system)
-    mock_future = create_real_future(test_template_success.install.return_value)
-    mock_executor.return_value.__enter__.return_value.submit.return_value = mock_future
-
-    result = installer.install([test_template_success])
-
-    assert result.success
-    assert result.message == "All test templates installed successfully."
-
-    # Check if the template is installed
-    assert installer.is_installed([test_template_success])
-
-
-@patch("cloudai._core.base_installer.ThreadPoolExecutor", autospec=True)
-def test_install_failure(mock_executor: Mock, slurm_system: SlurmSystem, test_template_failure: Mock):
-    installer = BaseInstaller(slurm_system)
-    mock_future = create_real_future(test_template_failure.install.return_value)
-    mock_executor.return_value.__enter__.return_value.submit.return_value = mock_future
-
-    result = installer.install([test_template_failure])
-
-    assert not result.success
-    assert result.message == "Some test templates failed to install."
-
-
-@patch("cloudai._core.base_installer.ThreadPoolExecutor", autospec=True)
-def test_uninstall_success(mock_executor: Mock, slurm_system: SlurmSystem, test_template_success: Mock):
-    installer = BaseInstaller(slurm_system)
-    mock_future = create_real_future(test_template_success.uninstall.return_value)
-    mock_executor.return_value.__enter__.return_value.submit.return_value = mock_future
-
-    result = installer.uninstall([test_template_success])
-
-    assert result.success
-    assert result.message == "All test templates uninstalled successfully."
-
-
-@patch("cloudai._core.base_installer.ThreadPoolExecutor", autospec=True)
-def test_uninstall_failure(mock_executor: Mock, slurm_system: SlurmSystem, test_template_failure: Mock):
-    installer = BaseInstaller(slurm_system)
-    mock_future = create_real_future(test_template_failure.uninstall.return_value)
-    mock_executor.return_value.__enter__.return_value.submit.return_value = mock_future
-
-    result = installer.uninstall([test_template_failure])
-
-    assert not result.success
-    assert result.message == "Some test templates failed to uninstall."
-
-    # Check if the template is still installed
-    assert installer.is_installed([test_template_failure])
-
-
```

Comment:
Could you please let me know why we are deleting these tests?

Reply:
These tests are for some reason duplicated in `tests/test_base_installer.py`